src/lib/Libpython/pbs_python_svr_external.c

```diff
@@ -584,10 +584,7 @@ pbs_python_do_vnode_set(void)
 void
 pbs_python_set_interrupt(void)
 {
-
-#ifdef PYTHON
-	PyErr_SetInterrupt();
-#endif
+	return;
```

Comment:
Without PyErr_SetInterrupt(), the signal never gets sent to the python interpreter.

Reply:
Yes you are right. Removing this does not fix the problem. I'll close this pull request.